content/concepts/introduction-to-webpack.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Introduction
+---
+
+*Webpack* is a module system for modern Javascript applications.
+
+In [modular programming](https://en.wikipedia.org/wiki/Modular_programming), developers break programs up into discrete chunks of functionality.
+This has a number of benefits.
+Each module has a smaller surface area for testing than a full program, making verification and bug-hunting easier.
+Well-written modules provide solid abstractions and encapsulation boundaries, so that each module has a coherent design and a clear purpose within the overall application.
+
+Node has supported modular programming almost since its inception.
+On the web, however, support for modules has been slow to arrive.
+Multiple tools exist that support modular Javascript on the web, with a variety of benefits and limitations.
+Webpack builds on lessons learned from these systems.
+
+In a Webpack application, each _module_ contains all of the code necessary to execute a single aspect of the application.
+Modules do not need to be self-contained.
+Code provided by one module can be reused in a second module: the first module _depends on_ the second module.
```

Comment:
Maybe change this to something like "Functionality provided by one module can be reused in various other modules. These other modules are said to _depend_on_ this module." You have got the first and second module mixed up in the dependencies here. 
Then maybe provide a relatable example here, something that is commonly used by everybody or a pattern that is commonly followed by everybody. Something like "momentjs is used by many applications to format time. Hence the applications are said to _depend_on_ momentjs as an external dependency"